tests/class-all-test.php

```diff
@@ -48,6 +49,7 @@ public function setUp() {
 			'lowercase_tags'            => true,
 			'track_post_types'          => array( 'post' ),
 			'track_page_types'          => array( 'page' ),
+			'logo'                      => '',
```

Comment:
Without this value, there will be a warning regarding this line. 
https://github.com/Parsely/wp-parsely/blob/ffc769f1858e886d9aded9e1a36c691d614add61/src/class-parsely.php#L1009-L1009

I think that's because the default value here is not the same as the main class
https://github.com/Parsely/wp-parsely/blob/ffc769f1858e886d9aded9e1a36c691d614add61/src/class-parsely.php#L33